packages/rum/src/domain/record/observers.ts

```diff
@@ -226,13 +229,19 @@ export function initMouseInteractionObserver(
   }).stop
 }
 
-function computeCoordinates(event: MouseEvent | TouchEvent) {
+function tryComputeCoordinates(event: MouseEvent | TouchEvent) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** small naming improvement‏
```suggestion
function tryToComputeCoordinates(event: MouseEvent | TouchEvent) {
```